### PR TITLE
Feature/as 3 extra conditions

### PR DESCRIPTION
**Pull request by davehlewis**
## Ticket Number

AS-3

## Description of change
Create Extra conditions question as per AS-3 ticket

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [X] My commit history in this PR is linear
- [X] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: lpa-submissions-e2e-tests/wait.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const fetch = require("node-fetch");
+
+console.log('Waiting for ZAP');
```
Comment (Fenisay) [inline lpa-submissions-e2e-tests/wait.js:3]: An omission perhaps?

**Review by Fenisay — DISMISSED** [review]
LGTM !

File: lpa-submissions-e2e-tests/wait.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+const max_wait = 2000;
+
+async function wait(ms) {
+    return new Promise(resolve => {
+        setTimeout(resolve, ms);
+    });
+}
+
+['http://localhost:8080/JSON/pscan/view/recordsToScan'].map(async (url) => {
```
Comment (danWatkinson) [inline lpa-submissions-e2e-tests/wait.js:13]: looks like a leftover bit of the zap stuff?
as an aside- i'd like to catch up with people about this wiring as i think there are much lighter touches available.. that said i would like to understand what all this is about as it feels like this is trying to wire the scanning into cypress to get it as part of the existing cypress output?

Comment (davehlewis) [reply lpa-submissions-e2e-tests/wait.js:13]: I think this is a discussion we need to have between teams for sure. This work was being led by @shilpabasu22 I believe so best to reach out to her

File: lpa-submissions-e2e-tests/wait.js (hunk 1 above)
Comment (davehlewis) [reply lpa-submissions-e2e-tests/wait.js]: Thanks I have removed this now - can you re-review?

**Review by Fenisay — APPROVED** [review]
LGTM

**Pull request merged**